src/audio/copier/copier.c

```diff
@@ -578,45 +523,39 @@ static int do_multi_endpoint_copy(struct copier_data *cd, struct comp_dev *dev)
 	return ret;
 }
 
-static int do_module_copy(struct copier_data *cd, struct comp_dev *dev)
-{
-	struct comp_buffer *src, *sink;
-	struct comp_buffer __sparse_cache *src_c, *sink_c;
-	struct comp_copy_limits processed_data;
-	struct list_item *sink_list;
-	int ret  = 0;
+static int do_module_process_copy(struct copier_data *cd, struct comp_dev *dev,
+				  struct input_stream_buffer *input_buffers,
+				  int num_input_buffers,
+				  struct output_stream_buffer *output_buffers,
+				  int num_output_buffers)
 
-	processed_data.source_bytes = 0;
+{
+	struct audio_stream __sparse_cache *source_s;
+	struct audio_stream __sparse_cache *sink_s;
```

Comment:
what does the `_s` suffix stand for? We usually use `_c` for "cached," what do you mean here?

Reply:
When I made the change, source_c and sink_c was occupied, so use _s, means stream cached.
I can change back, this is not big deal, current blocker is how to call copier trigger, may need a new interface or add it into copier, it may cause extra cycles.